content/blog/from-cms-to-contextual.mdx

```diff
@@ -6,7 +6,7 @@ author: James Perkins
 
 Tina allows you as a developer to create an amazing editing experience. By default the editing experience is a more traditional CMS, where you login to a specific URL and you edit your content without being able to see the content till after you finish your changes. But, if you are looking for a more transparent real-time editing experience, Tina has a superpower, called Visual Editing. Just as it sounds, you get instant feedback on the page as well as being able to preview the changes before publishing live to your site.
 
-![Example of Tina Context editing](https://res.cloudinary.com/forestry-demo/video/upload/q_100/v1647436971/blog-media/cms-to-contextual/TinaSuper.gif)
+<WebmEmbed embedSrc="/img/blog/from-cms-to-contextual/TinaSuper.webm"/>
```

Comment:
**nitpick**: Are we able to get a better WebM conversion for this animation?

The lack of anti-aliasing is obvious and makes the animation look less professional.

![image](https://github.com/user-attachments/assets/71bff2d5-a8bf-4efd-bef5-12c17b27b429)
**Figure: The WebM conversion on the left, the old GIF on the right, with the screenshot taken at 100% magnification both windows - note the ragged edges on the title text**

Reply:
Reviewed with Landon - the quality of the .webm matches the quality of the .gif but at 1/10 of the size

Reply:
Spoke with @landonmaxwell - its a render issue with the scaling on its render. Although the component `WebmEmbed` doesnt directly dictate anything to do with scaling.